src/helpers/filters.ts

```diff
@@ -96,6 +96,10 @@ export const filterBySearch = (search: string, transactions: Transaction[]) =>
     );
   });
 
+export const filterByStatus = (transactions: Transaction[], status: string) => {
```

Comment:
not sure if TypeScript figures it out automatically , but would be worth to add return type annotation
```
): Transaction[] => {
```